docs/workflows/authenticator-types.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Authenticator Types
+sidebar_position: 4
+---
+
+# Authenticator Types
+
+## Prerequisites
+
+- Set up a [developer account](./account-setup.md)
+
+## Overview
+
+Beyond Identity currently offers two different types of authenticators:
+
+- [Hosted Web Authenticator](#hosted-web-authenticator)
+- [Embedded SDK Authenticator](#embedded-sdk-authenticator)
+
+These can be configured in your [Authenticator Config](../platform-overview/authenticator-config.md).
+
+## Hosted Web Authenticator
+
+The hosted web authenticator is a webpage hosted by Beyond Identity that embeds the [JavaScript SDK](./sdk-setup.mdx). Think of this as a wallet, hosted by Beyond Identity Cloud, that can be used to hold an identity's passkeys.
+
+It can be found at `https://auth-<REGION>.beyondidentity.com/authenticator`.
+
+- https://auth-us.beyondidentity.com/authenticator
+- https://auth-eu.beyondidentity.com/authenticator
+
+The hosted web authenticator requires less configuration and code, but it does require redirecting your users to the hosted webpage. Keep in mind that passkeys are tied to an origin and they are stored on a specific browser.
+
+## Embedded SDK Authenticator
+
+The embedded SDK authenticator is embedded directly in your application on installation of an [SDK](./sdk-setup.mdx) (iOS, Android, Flutter, React Native, or JS).
+
+Using one of the SDKs requires more configuration and code, but it comes with more flexibility and passkeys are stored on the same device as your application.
```

Comment:
Since we're saying it comes with more flexibility, is it possible to provide an example or two of what the developer might want to change/optimize using the embedded authenticator? Basically what's the benefit?
Also, should "same device" be "same origin" as in the text above?